repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -101,3 +108,9 @@ class HeaderTranslationFilter @Inject()(configurationService: ConfigurationServi
     initialized = true
   }
 }
+
+object HeaderTranslationFilter {
+  private final val DefaultConfig = "header-translation.cfg.xml"
+  private final val SchemaFileName = "/META-INF/schema/config/header-translation.xsd"
+  private final val QualityReqex = ";q=\\d[^;]*".r
```

Comment:
Oh, it's totally Safe Enough. But why even enforce the first digit? It seems like a quirky mismatch of caring, but not that much.